backend/mobile-backend/schema.graphql

```diff
@@ -0,0 +1,58 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } } # FOR TESTING ONLY!
+
+type GameSession @model {
+  id: ID! @primaryKey(sortKeyFields: ["updatedAt"])
+  game: Game! @hasOne
+  startTime: AWSDateTime
+  phaseOneTime: Int
+  phaseTwoTime: Int
+  teams: [Team] @hasMany
+  updatedAt: AWSDateTime!
+}
+
+type Game @model {
```

Comment:
IMO `subscription` can be `null` for this and the `Question`

Also for both, we can have `timestamp` to be `null` as well